next/components/forms/widget-components/DateTimePicker/TimeSelector.tsx

```diff
@@ -69,9 +76,13 @@ const TimeSelector = ({
 
   useDidMount(() => {
     const hoursItemOffset =
-      hoursItemRef && hoursItemRef.current ? hoursItemRef.current[+hour]?.offsetTop : 0
+      hoursItemRef && hoursItemRef.current
+        ? hoursItemRef.current[+timeFormatArray[0]]?.offsetTop
+        : 0
```

Comment:
missed this on last review - but again, this is much nicer with ?.


```suggestion
      hoursItemRef?.current?.[+timeFormatArray[0]]?.offsetTop || 0
```